### PR TITLE
records: marcxml mapping improvement

### LINKED ISSUE #501: records: finish marcxml mapping
See #500 

Full marc example: 
https://github.com/zenodo/zenodo/blob/production/zenodo/base/testsuite/test_jsonext.py#L31-L206

### PR DESCRIPTION
**Pull request by hachreak**
* Improves marcxml mappings. (closes #501)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: tests/unit/records/test_schemas_marcxml.py [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+            "scheme": "ads",
+            "relation": "alternateIdentifier",
+        },
+        {
+            "identifier": "10.1234/alternate.doi",
+            "scheme": "doi",
+            "relation": "alternateIdentifier",
+        },
+    ]
+    full_record['meetings'] = {
```
Comment (lnielsen) [inline tests/unit/records/test_schemas_marcxml.py]: I think this should be a list with a dict inside.

Please add a test to see that it validates. E.g.:
``assert Record(full_record).validate()``

File: tests/unit/records/test_schemas_marcxml.py [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+        'title': 'The 13th Biennial HITRAN Conference',
+        'place': 'Harvard-Smithsonian Center for Astrophysics',
+        'dates': '23-25 June, 2014',
+        'acronym': 'HITRAN13',
+        'session': 'VI',
+        'session_part': '1',
+    }
+    full_record['authors'] = [
```
Comment (lnielsen) [inline tests/unit/records/test_schemas_marcxml.py]: authors -> creators

File: tests/unit/records/test_schemas_marcxml.py [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+        "oai": "oai:zenodo.org:1",
+        "indicator": ["user-zenodo", "user-ecfunded"]
+    }
+    serializer = JSONSerializer(RecordSchemaMARC)
+    data = serializer.serialize(
+        pid=PersistentIdentifier(pid_type='recid', pid_value='2'),
+        record=Record(full_record))
+    expected = {
+        u'terms_governing_use_and_reproduction_note': {
+            u'uniform_resource_identifier': u'http://zenodo.org',
```
Comment (lnielsen) [inline tests/unit/records/test_schemas_marcxml.py]: Where is this value coming from?

File: tests/unit/records/test_schemas_marcxml.py [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+            u'miscellaneous_information': u'HITRAN13',
+            u'name_of_part_section_of_a_work': u'1',
+            u'location_of_meeting':
+            u'Harvard-Smithsonian Center for Astrophysics'
+        },
+        'main_entry_personal_name': [
+            {
+                u'affiliation': u'CERN', u'personal_name': u'Doe, John',
+                u'authority_record_control_number_or_standard_number': [
+                    u'(gnd)gnd:170118215', u'(orcid)0000-0002-1694-233X'
```
Comment (lnielsen) [inline tests/unit/records/test_schemas_marcxml.py]: Expected:
``(gnd)gnd:170118215`` -> ``(gnd)170118215``

File: tests/unit/records/test_schemas_marcxml.py (hunk 3 above)
Comment (hachreak) [inline tests/unit/records/test_schemas_marcxml.py]: from here: https://github.com/zenodo/zenodo/blob/production/zenodo/base/testsuite/test_jsonext.py#L150

**Comment by hachreak** [comment]
ping @lnielsen

File: tests/unit/records/test_schemas_marcxml.py (hunk 3 above)
Comment (lnielsen) [inline tests/unit/records/test_schemas_marcxml.py]: Got it

**Pull request merged**